ssg/build_yaml.py

```diff
@@ -1498,6 +1531,9 @@ def to_file(self, file_name):
         tree = ET.ElementTree(root)
         tree.write(file_name)
 
+    def __str__(self):
+        return self.id_
```

Comment:
Do we need a string representation of a rule? A human-readable representation would be rather its title, and a reliable representation is basically its yaml.

Reply:
@matejak this is needed for utils/controls-template.html